src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ReadEnvelopesControllerTest.java

```diff
@@ -128,6 +128,14 @@ public void should_not_accept_invalid_statuses_when_reading_envelopes_by_status(
     private List<EnvelopeResponse> envelopesInDb() throws Exception {
         Envelope envelope = EnvelopeCreator.envelope("BULKSCAN", Status.PROCESSED);
         envelope.setZipFileName("7_24-06-2018-00-00-00.zip"); // matches expected response file
+        // todo: make this better :(
+        envelope.getScannableItems().forEach(item -> {
+            if (item.getOcrData() == null) {
+                item.setDocumentControlNumber("1111001");
+            } else {
+                item.setDocumentControlNumber("1111002");
```

Comment:
do those string have a meaning? If they do, maybe a comment would help

Reply:
i've already made a todo comment to look it over. of course some meaning exists. like "hardcoded response against actual values are compared should match" and dcn is now dynamically generated in `EnvelopeCreator.envelope`. better to look it over separately, iirc it has unused methods too

Reply:
Since this is going to be done anyway, how about solving `EnvelopeCreator.envelope` in another PR and rebasing this PR on that change?

Reply:
This change is fairly serious and was requested to be done asap. I'll mute further comments on that matter. Should not be too specific as all PRs we have but like I said - I'll mute

Making test framework PR changes as priority is a bit too eager at this point. But understandable

Reply:
Fair enough. For the time being, it would be worth adding a short comment explaining the intention behind the values.

Reply:
isn't `// todo` comment enough? i'd rather review the whole change implicated here instead of keep wondering "why this or that has some values i have no idea even though i myself implemented them" ;)